content/design/line-styles/index.html

```diff
@@ -2,65 +2,136 @@
+    <div class="definition-item">
+      <div class="bb-subsection-heading" style="border-bottom: 1px solid #cdcfd2;">Dark separator</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>The dark separator is a moderate gray, single-pixel line. This heavier stroke holds up between two different colors, which makes it ideal for borders and boundaries. It also frequently separates active content from passive content. For example, it can separate a toolbar from the content that the toolbar acts on.</div>
+            <p>SKY UX uses this style for:</p>
+            <ul>
+              <li>Borders on <a routerLink="../../components/button">buttons</a></li>
+              <li>Top and bottom boundaries for toolbars in areas such as <a routerLink="../../components/tiles">tiles</a> and <a routerLink="../../components/grid">grids</a></li>
+              <li>Column dividers for header rows in <a routerLink="../../components/grid">grids</a></li>
+              <li>Borders for <a routerLink="../../components/card">cards</a></li>
+              <li>Built-in data visualization axes</li>
+            </ul>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>@include bb-border(dark)</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>1px solid #cdcfd2</div>
+      </div>
+    </div>
+  </div>
 
----
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="bb-subsection-heading" style="border-bottom: 1px dotted #cdcfd2;">Row separator</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>This line style separates members of a homogenous set. It separates the rows in <a routerLink="../../components/grid">grids</a> or <a routerLink="../../components/repeater">repeaters</a>. The dotted style lowers its visual weight, and the moderate gray allows it to work with or without row striping on the list rows. It stretches to the full width of the rows (not the width of the container that holds the rows).</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>@include bb-border(row)</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>1px dotted #cdcfd2</div>
+      </div>
+    </div>
+  </div>
+  
+<h2 class="bb-section-heading" id="corners">Corners</h2>
 
-<h1 class="bb-page-heading">Line styles</h1>
+<p>In general, the style of corners should be handled by the component. If you need to directly specify the corners of an element, the rule of thumb is to use rounded corners for interactive elements and square corners for containers.</p>
```

Comment:
I'd change the first sentence to use active voice: "In general, the component should handle the style of corners."

Also, do we need an entry in this section for square corners?